flask_app/templates/view_profile.html

```diff
@@ -1,139 +1,113 @@
-                            <p>Athlete | Weight Loss | Strength</p>
-                            <!-- <p>
+                            <div class="card-title">
+                                <img src="../static/img/usericon.jpeg" alt="usericon" class="avatar-m">
+                            </div>
+                            <div class="card-body">
+                                <h1>Profile {{user.first_name}} {{user.last_name}}</h1>
+                                <h3>Social Fitness Life</h3>
+                                <p>Athlete | Weight Loss | Strength</p>
+                                <!-- <p>
                                 <a href="/edit_workout/">
                                     <img src="../img/gear.png" alt="gear" class="icon-s">
                                     Edit Workout
                                 </a>
                             </p> -->
+                            </div>
                         </div>
-                    </div>
-
-                    <div class="card">
-                        <div class="header">
-                            All Workouts
-                        </div>
-                        <table class="table table-hover table-light table-responsive form-control-md">
-                            <thead>
-                                <tr>
-                                    <th>Workout</th>
-                                    <th>Date</th>
-                                    <th>Actions</th>
-                                </tr>
-                            </thead>
-                            <tbody>
-
-                                {% for workout in user_workouts %}
-                                <tr>
-                                    <!-- {% for user in users %}
-                                    {% if user.id == workout.user_id %}
-                                    <td>{{user.first_name}}</td>
-                                    {% endif %}
-                                    {% endfor %} -->
-                                    <td>{{workout.type}}</td>
-                                    <td>{{workout.date.strftime('%B, %d, %Y')}}</td>
-                                    <td>
-                                        <a href="/view/workout/{{workout.id}}">View</a>
-                                        <a href="/edit_workout/{{workout.id}}">Edit</a>
-                                        <a href="/delete_workout/{{workout.id}}">Remove</a>
-                                    </td>
-                                </tr>
-                                {% endfor %}
-                            </tbody>
-                        </table>
-                    </div>
-
-                </div>
-
-                <!-- right column -->
-                <div class="col">
-                    <div class="card">
-                        <div class="header">
-                            <span class="friends">{{pending|length}}</span>
```

Comment:
We need these lines for the profile to function correctly!!! ahhhh i cant find them